### ISSUE TITLE
Implement keyboard backlight control

### ISSUE THREAD

**Issue by jekhor** [jekhor/yogabook-linux#8]

**Comment by jekhor** [comment]
Keyboard uses PWM2 for backlight control. Driver has been enabled in the https://github.com/jekhor/yogabook-linux-kernel/commit/e97674268bb047a461a0a4f844798d29ed16de1b

How to activate:
```
cd /sys/class/pwm/pwmchip1/
echo 0 > export
cd pwm0
echo 1 > enable
```

**Comment by Kruemmelspalter** [comment]
Hello 👋, is there a chance / is it in principle possible/reasonable that this patch will be mainlined?

**Comment by jekhor** [comment]
It is mainlined already. For YB1-X91 it should work for several years, for YB1-X90 it will be available in 7.0 kernel

**Comment by Kruemmelspalter** [comment]
ah kk thank you; did not realize it was already coming for 7.0
